6.x/base-how-to.md

```diff
@@ -413,3 +413,71 @@ Add whatever validation rules & inputs you want, in addition to name and passwor
 ```
 This will make the registration process pick up a view you can create, in ```resources/views/vendor/backpack/ui/auth/register.blade.php```. You can copy-paste the original view, and modify as you please. Including adding your own custom inputs.
 
+<a name="enable-email-verification-in-backpack-routes"></a>
+### Enable email verification in Backpack routes
+
+In Backpack CRUD 6.2 we introduced the ability to require email verification when accessing Backpack routes.
+
+To enable this feature please do the following: 
+
+**Step 1** - Ensure your Authenticable model (usually `App\Models\User`) implements the `Illuminate\Contracts\Auth\MustVerifyEmail` contract.
```

Comment:
```suggestion
**Step 1** - Make sure your user model (usually `App\Models\User`) implements the `Illuminate\Contracts\Auth\MustVerifyEmail` contract. [More info](https://laravel.com/docs/10.x/verification#model-preparation).
```

Reply:
![image](https://github.com/Laravel-Backpack/docs/assets/7188159/48786b39-5c31-4a61-bcde-e135778ed4f8)

I agree with all the rest tho.

Reply:
Exactly. We're not formal here. We shouldn't be. We're developers talking to developers.